app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<section id='recipe' class="container mx-auto px-4">
```

Comment:
- [ ] Good job so far but I have an issue with your design. I noticed that your recipe details don't follow the project wireframe. Please fix it in order to make your app design follows the wireframe 👍 

#### Your design
![recipe_details](https://github.com/Solobarine/recipes/assets/6615394/38dac7b5-de95-4498-8cf0-2c24f6ad56cd)

#### Wireframe
![recipe_food](https://github.com/Solobarine/recipes/assets/6615394/40555c0e-4b25-4484-88a7-0a23b7b668a4)